src/Uno.UWP/UI/Notifications/IBadgeUpdaterExtension.skia.cs

```diff
@@ -0,0 +1,9 @@
+﻿#nullable enable
```

Comment:
I think you will have to remove the "skia" suffix if you want to have it on wasm and macOS

Reply:
This one is OK, as it is just internal extension of `BadgeUpdater` on Skia